src/main/java/codeu/model/data/Message.java

```diff
@@ -72,4 +94,18 @@ public int getMessageLength(){
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Returns image associated with this Message */
+  public Blob getImage(){
```

Comment:
If you want to be fancy, Java 8 has a type `java.util.Optional<Blob>` that encapsulates the "may be null" feature.